scripts/monitoring/notify.py

```diff
@@ -0,0 +1,76 @@
+import json
+import time
+from typing import Callable, Optional, TypedDict
+
+import requests
+from typing_extensions import NotRequired
+
+
+timeout_between_sends = 61
```

Comment:
Per PEP8, constants should be all uppercase. Please also consider making this private if this is not being used outside of this module.